src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/Which.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tools.BootstrapTasks
+{
+	public class Which : Task
```

Comment:
Perhaps? The nice thing about `which` is that it shares the name of [the command-line program **which**(1)(http://linux.die.net/man/1/which), which is a common-enough utility that even [Windows developers ask about it](http://stackoverflow.com/questions/304319/is-there-an-equivalent-of-which-on-the-windows-command-line).

However, perhaps `Where` would be better, given the above answer?